frontend/src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Fab, TextField, Radio, RadioGroup, FormControlLabel, FormControl, InputAdornment, Box, Avatar, Typography } from '@mui/material';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import boy from '../../img/avatars/boy.png';
+import girl from '../../img/avatars/girl.png';
+import '@fontsource/roboto/500.css';
+import './Profile.scss';
+
+
+function Profile()  {
+    const [height, setHeight] = React.useState();
+    const [weight, setWeight] = React.useState();
+    const [bmi, setBmi] = React.useState(0);
+    const [gender, setGender] = React.useState('');
+    const [avatar, setAvatar] = React.useState(boy);
+    const [image, setImage] = React.useState('')
```

Comment:
I think those states (height, weight, etc.) needs to be available globally in our app, so we need to move them to App component or use useContext hook

Reply:
to avoid prop drilling (if those data should be available in multiple components) we should use context.
Little code snippet 
```import { createContext, useState } from "react";
export const BabyContext = createContext();
export const BabyProvider = ({ children }) => {
  const [height, setHeight] = useState(0);
  return (
    <BabyContext.Provider value={{ height, setHeight }}>
      {children}
    </BabyContext.Provider>
  );
};```